From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: BTC Core <core@borntocraftstudio.net>
Date: Fri, 14 Aug 2026 15:49:01 +0200
Subject: [PATCH] BTC-CORE hooks


diff --git a/src/main/java/org/bukkit/GameRules.java b/src/main/java/org/bukkit/GameRules.java
index fb6de66affb3c1b22632da67f9bb01a098591d6e..a83271f38f799775ea2fe6b7aa71cdc99c0a8d5c 100644
--- a/src/main/java/org/bukkit/GameRules.java
+++ b/src/main/java/org/bukkit/GameRules.java
@@ -26,6 +26,7 @@ public final class GameRules {
 
     public static final GameRule<Boolean> COMMAND_BLOCKS_WORK = getRule("command_blocks_work");
 
+    public static final GameRule<Integer> COPPER_FADE = getRule("copper_fade"); // BTCCore - copper oxidation rate
     public static final GameRule<Boolean> DROWNING_DAMAGE = getRule("drowning_damage");
 
     public static final GameRule<Boolean> ELYTRA_MOVEMENT_CHECK = getRule("elytra_movement_check");
